README.md

```diff
@@ -1,2 +1,154 @@
-# bookstore
-A bookstore website implemented in react and redux
+# Bookstore
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [Bookstore](#bookstore)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Javascript-Capstone-Group-Project\] ](#-bookstore-)
+  - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage:](#usage)
+    - [Run tests:](#run-tests)
+    - [Deployment:](#deployment)
+    - [Demo](#video demo)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 [MathMagicin] <a name="about-project"></a>
+
+**[MathMagician]** is a project that aims to build a dynamic bookstore website using the full features of React and Redux
+
```

Comment:
- [ ] I can see that your README still contains **MathMagician** your previous project. Please ensure that your README only contains details pertaining to the current project.